apps/condo/domains/meter/schema/MeterReading.js

```diff
@@ -145,6 +150,18 @@ const MeterReading = new GQLListSchema('MeterReading', {
                 update: access.canEditBillingStatusFields,
             },
         },
+        accountNumber: {
```

Comment:
Is it possible to make this field readonly?
```js
access: { read: true, create: true, update: false }
```

Reply:
yes, fixed it, thank you! it's much more concise